### PR TITLE
Fix #36058 sims4pack.ru

### LINKED ISSUE #36058: sims4pack.ru
### Issue URL (Ads)
[https://sims4pack.ru/obuv-dlya-sims-4/4015-Baletki-Nidia-dlya-Sims-4/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fsims4pack.ru%2Fobuv-dlya-sims-4%2F4015-Baletki-Nidia-dlya-Sims-4%2F)
### Comment
Username: @Evaanna
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | Ext
AdGuard version: | 3.1.1
Browser: | YaBrowser
Stealth mode options: | Hide your search queries, Send Do-Not-Track header, Strip URLs from tracking parameters, Self-destructing third-party cookies (2880), Hide your Referrer from third-parties
Filters: | AdGuard Russian,<br>Filter unblocking search ads and self-promotion,<br>AdGuard Annoyances,<br>AdGuard Base,<br>AdGuard Tracking Protection,<br>AdGuard Social Media

### PR DESCRIPTION
**Pull request by tvinzz**
https://github.com/AdguardTeam/AdguardFilters/issues/36058

### CODE REVIEW

File: SocialFilter/sections/specific.txt [hunk 1]
```diff
@@ -24,6 +24,9 @@ citydog.by##.headerBlockShare
 don24.ru##.socials-and-print
 newser.com###divSocialAndCommentsButtonsTop > a:not([id="aShareComments"])
 fishki.net##.content__figure__share__social
+||vk.com/widget_community.php^$domain=sims4pack.ru,third-party
+sims4pack.ru##.menu-block:has(#vk_groups)
```
Comment (BlazDT) [inline SocialFilter/sections/specific.txt]: Please move this and the following rule to css_extended. Also please modify both ext-css rules to include parent element:

.block-right > .menu-block

File: SpywareFilter/sections/specific.txt [hunk 2]
```diff
@@ -18,6 +18,11 @@
 kinotan.ru##.counts
 ||fstrk.net/api/tracker/
 ||massdrop.com/*/pixelrb?
+||yastatic.net^$domain=sims4pack.ru,third-party
+||mc.yandex.ru^$domain=sims4pack.ru,third-party
```
Comment (BlazDT) [inline SpywareFilter/sections/specific.txt]: This and the following domains are already blocked by Spyware filter.

File: SpywareFilter/sections/specific.txt [hunk 3]
```diff
@@ -18,6 +18,11 @@
 kinotan.ru##.counts
 ||fstrk.net/api/tracker/
 ||massdrop.com/*/pixelrb?
+||yastatic.net^$domain=sims4pack.ru,third-party
```
Comment (BlazDT) [inline SpywareFilter/sections/specific.txt]: Not spyware. Social filter already blocks share.js and script file blocked contains this:
https://github.com/es-shims/es5-shim

**Review by BlazDT — CHANGES_REQUESTED** [review]

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: SocialFilter/sections/css_extended.txt [hunk 4]
```diff
@@ -1,6 +1,8 @@
 !
 ! Extended CSS rules
 !
+sims4pack.ru##.block-right > .menu-block:has(#vk_groups)
```
Comment (Alex-302) [inline SocialFilter/sections/css_extended.txt]: better add `.block-right > .menu-block:has(> #vk_groups)`

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**